os_migrate/plugins/module_utils/user.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import openstack
+
+from ansible_collections.os_migrate.os_migrate.plugins.module_utils \
+    import const, reference, resource
+
+
+class User(resource.Resource):
+    resource_type = const.RES_TYPE_USER
+    sdk_class = openstack.identity.v3.user.User
+
+    info_from_sdk = [
+        'id',
+        'domain_id',
+        'default_project_id',
+    ]
+
+    params_from_sdk = [
+        'name',
+        'description',
+        'email',
+        'is_enabled',
+    ]
+
+    params_from_refs = [
+        'default_project_ref',
+    ]
+
+    sdk_params_from_refs = [
+        'default_project_id',
+    ]
+
+    @staticmethod
+    def _refs_from_sdk(conn, sdk_res):
+        refs = {}
+        refs['default_project_ref'] = reference.user_ref(
+            conn, sdk_res['default_project_id'])
+        refs['default_project_id'] = sdk_res['default_project_id']
+        return refs
+
+    def _refs_from_ser(self, conn, filters=None):
+        refs = {}
+        refs['default_project_ref'] = self.params()['default_project_ref']
+        refs['default_project_id'] = reference.user_id(
+            conn, self.params()['security_group_ref'])
```

Comment:
Same here, we need to use project_id function. Also, there's copypasta with "security_group_ref" here.